src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -19,10 +20,30 @@
 (react/defc GroupTable
   {:render
    (fn [{:keys [this state]}]
-     (net/render-with-ajax
-      (:groups-response @state)
-      #(this :-render-groups-table)
-      {:loading-text "Loading Groups..."}))
+     [:div {}
+      (cond
+        (:delete-modal? @state)
+        (if (:deleting? @state)
+          [comps/Blocker {:banner "Deleting Group"}]
+          (modals/render-message
+           {:text "Are you sure you want to delete this group?"
+            :on-confirm (fn []
+                          (swap! state assoc :deleting? true)
+                          (net/render-with-ajax
```

Comment:
@UrsaStutsman let me know when you're going to work on this and we can work through it together <3